### PR TITLE
testing/toxcore: testing/toxic: New package for Tox messenger

### PR DESCRIPTION
**Pull request by strfry**
This PR adds a package for the Tox Messenger console client 'toxic', and the underlying 'toxcore' library.

### CODE REVIEW

**Comment by strfry** [comment]
I just fixed a dependency problem and disabled the tests for toxcore (they require networking and don't work properly on Travis somehow).

**Comment by strfry** [comment]
I am still waiting for a review or response.

**Comment by clandmeter** [comment]
Can you please rebase this against master.

File: testing/toxcore/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+builddir="$srcdir/c-$pkgname-$pkgver"
+
+prepare() {
+	cd "$builddir"
+	cmake . \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_LIBDIR=lib \
+		|| return 1
```
Comment (clandmeter) [inline testing/toxcore/APKBUILD]: cmake needs to run in build.
There is also no need to return 1 please remove it.

**Comment by strfry** [comment]
stuff is rebased and fixed now

File: testing/toxcore/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+source="$pkgname-$pkgver.tar.gz::https://github.com/TokTok/c-toxcore/archive/v$pkgver.tar.gz"
+
+# Toxcore's tests do require networking and are not working properly in Travis-CI
+options="!check"
+
+builddir="$srcdir/c-$pkgname-$pkgver"
+
+prepare() {
+	cd "$builddir"
+	cmake . \
```
Comment (maxwell-k) [inline testing/toxcore/APKBUILD]: I think @clandmeter was trying to explain that `cmake` should be called inside the `build()` function, once you've moved the `cmake` command I don't think there is any need for a `prepare()` function at all you can probably remove the whole function definition and rely on abuild running the default automatically

**Comment by maxice8** [comment]
1 Month without response, can be closed ?

**Comment by strfry** [comment]
I'll merge the requested changes soon. Just now i don't use Alpine as a primary system anymore, and have to set up a build environment first to test.

**Pull request closed without merging**